src/backend/efiling-api/pom.xml

```diff
@@ -165,6 +166,7 @@
         <dependency>
             <groupId>redis.clients</groupId>
             <artifactId>jedis</artifactId>
+            <version>6.0.0</version>

```

Comment:
even after I add jedis.6.0.0 to BOM file,   jedis dependency goes back to 5.2.0.  so I have to include 6.0.0 here.